src/Files.App/Files.App.csproj

```diff
@@ -134,5 +134,9 @@
         <TrimmerRootAssembly Include="CommunityToolkit.WinUI.UI.Controls.Media" />
         <TrimmerRootAssembly Include="CommunityToolkit.WinUI.UI.Controls.Primitives" />
     </ItemGroup>
+
+	<ItemGroup>
+		<AdditionalFiles Include="Strings\en-US\Resources.resw" />
+	</ItemGroup>
```

Comment:
Files are included even without manually listing them here.
```suggestion
```

Reply:
If I don't specify a file for the source generator, it won't find it. The file must be added as AdditionalFiles. Do you have another way to do this without listing the file in AdditionalFiles?

Reply:
CsWin32 doesn’t seem to require this for NativeMethods.txt while the .NET banned API analyzer requires.